.github/workflows/azure-app-service-api.yml

```diff
@@ -57,5 +57,5 @@ jobs:
         with:
           app-name: ${{secrets.ApiAppName}}
           slot-name: 'Production'
-          publish-profile: ${{ secrets.AZUREAPPSERVICE_PUBLISHPROFILE_API }}
+          publish-profile: ${{ secrets.AZUREAPPSERVICE_PUBLISHPROFILE_035208C67C0748C9A736CE9EE26D4A28 }}
```

Comment:
I think this should be left as AZUREAPPSERVICE_PUBLISHPROFILE_API with a secret created from your app's publish profile (available on the overview page).  It will deploy normally but I think it can cause issues for the grader when they're going over it (not sure though, it might be safe to leave it).

Reply:
I didn't know that it made a difference. I'll change it back and see if it still works.